MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -275,6 +275,73 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <!-- Payara 5.202+ - Rest Client 1.4 -->
+            <id>payara5.202+</id>
+            <activation>
+                <property>
+                    <name>payara.version</name>
+                    <value>/5\.20[2-4].*/</value>
+                </property>
+            </activation>
+            <properties>
+                <microprofile.rest.client.tck.version>1.4.1.payara-p1</microprofile.rest.client.tck.version>
```

Comment:
Not your fault but the current setup does not define a version outside of a profile which at least in eclipse has the effect that the TCK dependency is not resolved in a way that the IDE searches and opens the TCK classes which is really annoying. What I did is define the TCK version property in the general `<properties>` block and changed the TCK dependency to scope `test`. This is also how most other MP TCK runners work so it would be nice to get this more uniform.

Reply:
Added some defaults